test/e2e/flowaggregator_test.go

```diff
@@ -65,8 +67,16 @@ DATA SET:
     destinationServicePortName:
     ingressNetworkPolicyName: test-flow-aggregator-networkpolicy-ingress
     ingressNetworkPolicyNamespace: antrea-test
+	ingressNetworkPolicyUID: ac42f0ba-f580-46ae-8336-2bfc45024a9a
+	ingressNetworkPolicyType: 2
+	ingressNetworkPolicyRuleName: test-ingress-rule-name
+	ingressNetworkPolicyRulePriority: 50000
```

Comment:
extra indent here

Reply:
have no idea why there is extra indent, which doesn't exist locally.
There are also extra indent in the main:
https://github.com/antrea-io/antrea/blob/c21592c0e718539f6a190843eb9fff9af8d3d175/test/e2e/flowaggregator_test.go#L70